src/Administration/Resources/app/administration/src/module/sw-plugin/view/sw-plugin-list/index.js

```diff
@@ -58,8 +58,13 @@ Component.register('sw-plugin-list', {
         },
 
         pluginCriteria() {
+            if (this.listingCriteria) {
+                return this.listingCriteria;
```

Comment:
what happen when the listingCriteria exists and the user changes the sortDirection or uses a search term?